packages/firestore/test/integration/api/database.test.ts

```diff
@@ -158,14 +160,17 @@ apiDescribe('Database', persistence => {
       const initialData = {
         untouched: true,
         foo: 'bar',
+        inner: { untouched: true, foo: 'bar' },
```

Comment:
inner looks identical to nested...  what am I missing?

Reply:
This is meant to match `can delete field using mergeFields` which specifies two different field depths for the field mask. I removed it from here since it doesn't make sense for this test.